index.html

```diff
@@ -0,0 +1,552 @@
+          Promise&lt;undefined&gt; disconnect(optional SmartCardDisposition disposition = "leave");
+
+          Promise&lt;ArrayBuffer&gt; transmit(BufferSource sendBuffer);
+
+          Promise&lt;any&gt; startTransaction(TransactionStartedCallback transaction);
+
+          Promise&lt;SmartCardConnectionStatus&gt; status();
+
+          Promise&lt;ArrayBuffer&gt; control([EnforceRange] unsigned long controlCode,
+              BufferSource data);
+
+          Promise&lt;ArrayBuffer&gt; getAttribute([EnforceRange] unsigned long tag);
+          Promise&lt;undefined&gt; setAttribute([EnforceRange] unsigned long tag, BufferSource value);
+        };
+
+        callback TransactionStartedCallback = Promise&lt;any&gt; ();
+      </pre>
+      <section>
+        <h3><dfn>disconnect()</dfn> method</h3>
+        <section data-dfn-for="SmartCardDisposition">
+          <h4><dfn>SmartCardDisposition</dfn> enum</h3>
+          <pre class="idl">
+            enum SmartCardDisposition {
+              "leave",
+              "reset",
+              "unpower",
+              "eject"
+            };
+          </pre>
+          <dl>
+            <dt><dfn>leave</dfn></dt>
+            <dd>Don't alter card state. SCARD_LEAVE_CARD in the [[PCSC5]] spec.</dd>
+            <dt><dfn>reset</dfn></dt>
+            <dd>Reset the card. SCARD_RESET_CARD in the [[PCSC5]] spec.</dd>
+            <dt><dfn>unpower</dfn></dt>
+            <dd>Unpower and terminate access to the card. SCARD_UNPOWER_CARD in
+              the [[PCSC5]] spec.</dd>
+            <dt><dfn>eject</dfn></dt>
+            <dd>Eject the card from the reader. SCARD_EJECT_CARD in the
+              [[PCSC5]] spec.</dd>
+          </dl>
+        </section>
+      </section>
+      <section>
+        <h3><dfn>transmit()</dfn> method</h3>
+      </section>
+      <section>
+        <h3><dfn>startTransaction()</dfn> method</h3>
+        <p>Starts a transaction. The Transaction is ended when the promise
+        returned by the given callback settles.</p>
+      </section>
+      <section>
+        <h3><dfn>status()</dfn> method</h3>
+        <p>Queries the host's PC/SC stack about the current connection status.</p>
+        <section data-dfn-for="SmartCardConnectionStatus">
+          <h4><dfn>SmartCardConnectionStatus</dfn> dictionary</h3>
+          <pre class="idl">
+            dictionary SmartCardConnectionStatus {
+              required DOMString readerName;
+              required SmartCardConnectionState state;
+              required ArrayBuffer answerToReset;
+            };
+          </pre>
+          <dl>
+            <dt><dfn>readerName</dfn> member</dt>
+            <dd>Name of the connected reader.</dd>
+            <dt><dfn>state</dfn> member</dt>
+            <dd>Current state of the connection.</dd>
+            <dt><dfn>answerToReset</dfn> member</dt>
+            <dd>The answer to reset (ATR) string from the card, if applicable.
+            Empty otherwise.</dd>
```

Comment:
Similar to above, in cases where the ATR is not applicable this should probably be `undefined` rather than an empty `ArrayBuffer`.

Reply:
Done.